src/components/tag-input/tag-input.component.ts

```diff
@@ -192,6 +192,9 @@ export class TagInputComponent<T = any> implements AfterContentInit, OnChanges,
     /** Raised when a tag has been clicked. The `tag` property of the event contains the clicked tag. Call `preventDefault()` on the event to prevent the default behaviour of selecting the tag. */
     @Output() tagClick = new EventEmitter<TagInputEvent>();
 
+    // When clicking on the input during mutliple mode it will send a on touched event to the parent component
+    @Output() onTouched = new EventEmitter<TagInputEvent>();

```

Comment:
It might be good to make this output called `inputFocus` instead. The `tag-input` component is a component that an app could use on it's own (not in the select) so `onTouched` is not really something that normally gets an event whereas the input focus could be something an app could be interested in.

As a side note, for naming outputs we generally want to avoid prefixing them with `on` (see Angular Styleguide https://angular.io/guide/styleguide#dont-prefix-output-properties).

We also try to avoid using `-ed` on the end of event names, for example if you look at the built in browser events Angular provides, the are `(click)`, `(focus)`, `(blur)` instead of `(clicked)`, `(focused)`, `(blurred)`

Reply:
It may also be good to add the `inputFocus` event to the doc? https://uxaspects.github.io/UXAspects/#/components/input-controls#tags